### ISSUE TITLE
ArrayIndexOutOfBoundsException error

### ISSUE THREAD

**Issue by KennyTC** [zezo357/pytorch_lite#15]
I have trained a custom Object Detection with yolov5, then I export it to .torchscript format. 
When I run the model with your example, just replacing  

- `yolov5s.torchscript` to `best.torchscript`
- `labels_objectDetection_Coco.txt` to `labels_objectDetection_digital.txt`

then, the following error happened. 
The same error appeared when I detected objects with camera or with images.
According to the error, I thought that there is something wrong with my labels in 'labels_objectDetection_digital.txt`, but I have checked it thoughtfully. It is a model to detect number from 0 to 9. 
I share the models and the labels here just in case you need. 

```
E/PytorchLitePlugin(12633): error classifying image
E/PytorchLitePlugin(12633): java.lang.ArrayIndexOutOfBoundsException: length=378000; index=378000
E/PytorchLitePlugin(12633): 	at com.zezo357.pytorch_lite.PrePostProcessor.outputsToNMSPredictionsYolov5(PrePostProcessor.java:202)
E/PytorchLitePlugin(12633): 	at com.zezo357.pytorch_lite.PrePostProcessor.outputsToNMSPredictions(PrePostProcessor.java:239)
E/PytorchLitePlugin(12633): 	at com.zezo357.pytorch_lite.PytorchLitePlugin.getImagePredictionListObjectDetection(PytorchLitePlugin.java:238)
E/PytorchLitePlugin(12633): 	at com.zezo357.pytorch_lite.Pigeon$ModelApi$-CC.lambda$setup$3(Pigeon.java:248)
E/PytorchLitePlugin(12633): 	at com.zezo357.pytorch_lite.Pigeon$ModelApi$$ExternalSyntheticLambda3.onMessage(Unknown Source:2)
E/PytorchLitePlugin(12633): 	at io.flutter.plugin.common.BasicMessageChannel$IncomingMessageHandler.onMessage(BasicMessageChannel.java:217)
E/PytorchLitePlugin(12633): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
E/PytorchLitePlugin(12633): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:319)
E/PytorchLitePlugin(12633): 	at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12)
E/PytorchLitePlugin(12633): 	at io.flutter.embedding.engine.dart.DartMessenger$SerialTaskQueue.flush(DartMessenger.java:173)
E/PytorchLitePlugin(12633): 	at io.flutter.embedding.engine.dart.DartMessenger$SerialTaskQueue.lambda$dispatch$0$io-flutter-embedding-engine-dart-DartMessenger$SerialTaskQueue(DartMessenger.java:163)
E/PytorchLitePlugin(12633): 	at io.flutter.embedding.engine.dart.DartMessenger$SerialTaskQueue$$ExternalSyntheticLambda0.run(Unknown Source:2)
E/PytorchLitePlugin(12633): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
E/PytorchLitePlugin(12633): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
E/PytorchLitePlugin(12633): 	at java.lang.Thread.run(Thread.java:1012)
```
[best.zip](https://github.com/zezo357/pytorch_lite/files/11048294/best.zip)
[labels_objectDetection_digital.txt](https://github.com/zezo357/pytorch_lite/files/11048304/labels_objectDetection_digital.txt)

**Comment by zezo357** [comment]
@KennyTC did you specify the model count?
```
 _objectModel = await PytorchLite.loadObjectDetectionModel(
          pathObjectDetectionModel, 80, 640, 640,
          labelPath: "assets/labels/labels_objectDetection_Coco.txt");
```

the 80 is the labels count

**Comment by KennyTC** [comment]
Wow, I did not notice that. It fixed now. Thanks a lot.
One more question: For object detection, the models needs to be converted to .torchscript? or it can work with .pt extension like classification task as well?

**Comment by zezo357** [comment]
It should work using torchscript only, but you can test the normal model and let me know if it works (I didn't test it 😅)

**Comment by KennyTC** [comment]
Well, it said '.pt is not a proper model'. Only classification task can use '.pt' extension, right? 
one more question: Does your library can run on MacOS?

**Comment by zezo357** [comment]
> Well, it said '.pt is not a proper model'. Only classification task can use '.pt' extension, right? 
> one more question: Does your library can run on MacOS?

Nor iOS or macos are supported for now, but I'm currently checking the ffi option to allow cross platform usage.


It may take sometime, but whenever I finish I will release a new version
